src/mc-fle2-insert-update-payload-private.h

```diff
@@ -68,10 +70,25 @@ typedef struct {
    bson_type_t valueType;                      // t
    _mongocrypt_buffer_t value;                 // v
    _mongocrypt_buffer_t serverEncryptionToken; // e
+   mc_array_t edgeTokenSetArray;               // g
    _mongocrypt_buffer_t plaintext;
    _mongocrypt_buffer_t userKeyId;
 } mc_FLE2InsertUpdatePayload_t;
 
+/**
+ * EdgeTokenSet is the following BSON document:
+ * d: <binary> // EDCDerivedFromDataTokenAndCounter
+ * s: <binary> // ESCDerivedFromDataTokenAndCounter
+ * c: <binary> // ECCDerivedFromDataTokenAndCounter
+ * p: <binary> // Encrypted Tokens
+ */
+typedef struct {
```

Comment:
Could this struct instance be inserted into `mc_FLE2InsertUpdatePayload_t`, since these tokens seem to travel around with each other in groups most of the time.